change/@uifabric-styling-2019-07-31-14-05-43-phkuo-leftNavRedlines.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Nav: update to latest redlines",
```

Comment:
The description here does not match to the changes to this specific package. It should probably be something along _Adds `bodyBackgroundHovered` and `bodyBackgroundChecked` semantic slots._